haconiwa-dev-company.yaml

```
@@ -0,0 +1,681 @@
+      - id: haconiwa-village
+        name: "Haconiwa Village"
+        companies:
+        - name: haconiwa-dev-company
+          grid: "8x4"
+          basePath: "./haconiwa-dev-company"
+          organizationRef: "haconiwa-dev-company-org"
+          agentDefaults:
+            type: "claude-code"
+            permissions:
+              allow:
+                - "Bash(python -m pytest)"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 16

(indentation)